README.md

```diff
@@ -25,19 +25,31 @@ RandomizedSearchCV was applied for hyperparameter optimization.
 The model was evaluated using metrics such as accuracy, precision, recall, and F1-score.
 
 ## Usage
-For the first time running the project, create the conda environment by running the following in the root of the repository:
 
-```bash
-conda-lock install --name wine-quality-regressor conda-lock.yml
-```
+### Setup
+
+> If you are using Windows or Mac, then please ensure that Docker Desktop is running.
```

Comment:
check if the user have docker
`docker --version`

Reply:
I will add that as a command the user can run to check if the user has Docker.